spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc

```diff
@@ -358,18 +358,20 @@ Here is a typical layout:
 ----
 	com
 	 +- example
-	     +- myproject
+	     +- myApplication
```

Comment:
Package names are usually all lower case. Please change this back to `myproject`.